### ISSUE TITLE
Water Mark positioning

### ISSUE THREAD

**Issue by Patrak59** [Polarrco/polarr_chrome_web_support#23]
Firstly, absolutely love your app, it's just what I need on my chromebook. Anyway, my issue is with the watermark feature. I recently edited the water mark and replaced it with my own, I proceeded to edit some photo's and saved them one at a time, with the water mark addition ticked. The position of the water mark was the bottom right hand corner of the photo. When I looked at my saved and edited photo's in my photo viewer, I noticed that some photo's had the water mark. just where I wanted but on other photo's, there was no water mark at all. The only difference between the photo's was that some of them I had cropped before saving them, would this have affected the water mark position?

**Comment by bwang29** [comment]
Hello, the cropping might cause the issue. Does removing cropping make the issue disappear?

**Comment by Patrak59** [comment]
Yes it is the cropping that causes the issue and that is my point. Why does
this happen because I crop the photo before saving and adding the water
mark. Is it possible to add a feature so that you can re-position the water
mark after cropping?
